tests/cases/item_test.py

```diff
@@ -352,6 +352,15 @@ def testItemMetadataCrud(self):
         self.assertStatusOk(resp)
         item = resp.json
 
+        # Try to delete metadata from an item that doesn't have any set on it
+        # yet.
+        resp = self.request(path='/item/%s/metadata' % (item['_id']),
+                            method='DELETE', user=self.users[0],
+                            body=json.dumps(['foobar']), type='application/json')
```

Comment:
I was following the style in the rest of this file. What would you like me to do here?